cmd/activator/main.go

```diff
@@ -237,12 +237,13 @@ func main() {
 	// NOTE: MetricHandler is being used as the outermost handler of the meaty bits. We're not interested in measuring
 	// the healthchecks or probes.
 	ah = activatorhandler.NewMetricHandler(env.PodName, ah)
+	// We need the context handler to run first so ctx gets the revision info.
```

Comment:
This is fine as we the handler that consumes the body is not run yet. The error is btw:

```
2023/11/25 00:07:42 httputil: ReverseProxy read error during body copy: read tcp 10.244.0.32:37488->10.244.0.31:8012: use of closed network connection
```
Even without the activator wrapper the error at the activator side happens vary rarely, unlike the QP side.